### PR TITLE
[KZL-181] As an end user, i'd like to use Kuzzle Admin Console on my desktop

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
:warning: Don't merge this PR yourself :warning: 

## What does this PR do ?
Add a simple electron packaging stage to Travis and push ZIP to dl.kuzzle.io

### How should this be manually tested?
Check https://dl.kuzzle.io and download the zipfile matching your platform.

### CODE REVIEW

**Comment by alexandrebouthinon** [comment]
WARNING : Do not merge this PR ! 
I use travis to publish the DoD on S3. So I need to update the travis.yml when all the reviewers accept this PR.

**Comment by alexandrebouthinon** [comment]
@Aschen: It's Ok for the zoom level and version number in package name. But version number in window title must be perform in VueJS code not in Electron.

File: .electron/main.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+let mainWindow;
+
+function createWindow () {
+  // Create the browser window.
+  mainWindow = new BrowserWindow({
+    width: 1280,
+    height: 720,
+    devTools: false,
+    webPreferences: {
+      zoomFactor: 0.8
```
Comment (xbill82) [inline .electron/main.js]: I think this is the DPI @benoitvidis is talking about: why not `1`?

Comment (alexandrebouthinon) [reply .electron/main.js]: Because, we need to scroll down to fill form pages. But I think have found a solution. I'll put the PR on WIP.

Comment (xbill82) [reply .electron/main.js]: it looks more like a CSS issue than something we should tweak on Electron, no?

Comment (alexandrebouthinon) [reply .electron/main.js]: Yes, you're right :disappointed:

**Pull request merged**